README.md

```diff
@@ -157,6 +157,29 @@ You can parse these or pass them directly into a command to run tests, like so:
 pub run build_runner test $(pub run test_html_builder:browser_aggregate_tests --mode=args [--release])
 ```
 
+### Randomizing the browser aggregation file
+By default, the tests will be aggregating in a specific order and this will be maintained between
+each test run. In order to mimic the dart test runner's built in test shuffling capability a consumer can add
+this to their yaml file to randomly shuffle the test order.
+
+```yaml
+targets:
+  $default:
+    builders:
+      test_html_builder:
+        options:
+          ...
+          randomize_ordering_seed: random
```

Comment:
We should include the `browser_aggregation: true` option here, too, since the randomization only applies to those aggregate tests. Speaking of which, we should probably warn if we detect that a package has configured a seed here but does not have browser aggregation enabled.